src/PaymentManager/PaymentManager.Repositories/Implementations/ReceivableRepository.cs

```diff
@@ -104,5 +104,10 @@ public async Task<ReceivableEntity> UpdateAsync(ReceivableEntity entity)
 
             return await GetByIdAsync(entity.AgencyId, entity.Id);
         }
+
+        public Task UpdateRange(List<PayableEntity> entities)
+        {
+            throw new NotImplementedException();

```

Comment:
Add `TODO` for not forget to implement. (Not forgetting to mention the date and author)